docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -102,14 +102,47 @@ echo $IAM_ROLE_ARN_FOR_IRSA
 
 For more information on authorization and access for ACK service controllers, including details regarding recommended IAM policies, see [Configure Permissions][configure-permissions].
 
+#### Optional step
+
+If you are planning to try out the Domain, UserProfile or App resource then `AmazonSageMakerFullAccess` won't be enough.
```

Comment:
nit: reword

`AmazonSageMakerFullAccess` policy does not include permissions required to manage Domain, UserProfile or App resource. Create and attach the following Policy to the IAM Role to ensure that your SageMaker service controller has access to these resources.

Reply:
thanks !